assets/js/modules/auth.js

```diff
@@ -0,0 +1,147 @@
+define([
+    'jquery',
+    'sourceModules/module'
+], function($, Module) {
+    
+    'use strict';
+
+    /**
+     * @Object default module option values
+     */
+    var defaults = {
+        'storageKey': 'sourcejsUser',
+        'defaultAvatarURL': '/source/assets/i/unknown.gif',
+        'classes': {
+            'controlsWrapper': 'sjs-login',
+            'loginButton': 'sjs-login-button',
+            'avatar': 'sjs-login-avatar',
+            'anonymous': 'anonymous'
+        },
+        'labels': {
+            'login': 'Login',
+            'logout': 'Logout'
+        }
+    };
+
+    /**
+     * @module Auth - basic GitHub authorization module.
+     *
+     * @constructor
+     *
+     * @param [Object] config - auth inline configuration set of options
+     *
+     * @param [Object] config.target - jquery domElement which goes to be auth controlls container
+     *
+     * @param [Object] config.options - options set, which allows to define component configuration.
+     */
+
+    function Auth(config) {
+        var _this = this;
+
+        this.conf = $.extend(true, {},
+            defaults,
+            config.options,
```

Comment:
I don't think so. This config is common for entire project (for team), options is used to customize local instance (single user).